### PR TITLE
Add `LiveServer.jl` as an option for running a local web server

### PR DESCRIPTION
**Pull request by DilumAluthge**
Currently, the docs only give a Python option for running a local web server.

We might as well also give a Julia option.

### CODE REVIEW

File: docs/src/man/guide.md [hunk 1]
```diff
@@ -141,7 +141,10 @@ build/
     not resolve directory URLs like `foo/` to `foo/index.html` for local files. You have two
     options:
 
-    1. You can run a local web server out of the `docs/build` directory. If you have Python
+    1. You can run a local web server out of the `docs/build` directory. One way to accomplish
+       this is to install the [LiveServer](https://github.com/asprionj/LiveServer.jl) Julia
+       package. You can then start the server with
+       `julia -e 'using LiveServer; cd("docs/build"); serve()'`. Alternatively, if you have Python
        installed, you can simple start one with `python3 -m http.server --bind localhost`
```
Comment (mortenpi) [inline docs/src/man/guide.md:148]:
```suggestion
       installed, you can start one with `python3 -m http.server --bind localhost`
```

**Comment by mortenpi** [comment]
Good idea to advertise it, thanks! LGTM.

File: docs/src/man/guide.md [hunk 2]
```diff
@@ -141,8 +141,11 @@ build/
     not resolve directory URLs like `foo/` to `foo/index.html` for local files. You have two
     options:
 
-    1. You can run a local web server out of the `docs/build` directory. If you have Python
-       installed, you can simple start one with `python3 -m http.server --bind localhost`
+    1. You can run a local web server out of the `docs/build` directory. One way to accomplish
+       this is to install the [LiveServer](https://github.com/asprionj/LiveServer.jl) Julia
+       package. You can then start the server with
+       `julia -e 'using LiveServer; cd("docs/build"); serve()'`. Alternatively, if you have Python
```
Comment (fredrikekre) [inline docs/src/man/guide.md:147]: `serve(dir="docs/build")` might be nicer.

Comment (DilumAluthge) [reply docs/src/man/guide.md:147]: https://github.com/JuliaDocs/Documenter.jl/pull/1385

**Pull request merged**